src/script/conversation/ConversationRepository.ts

```diff
@@ -2722,11 +2720,11 @@ export class ConversationRepository {
    * @returns Resolves when the event was handled
    */
   private async onMessageHidden(eventJson: MessageHiddenEvent) {
-    const {conversation: conversationId, data: eventData, from} = eventJson;
+    const {conversation, qualified_conversation, data: eventData, from} = eventJson;
```

Comment:
can we use camelCase here for the qualified_conversation to maintain consistency

Reply:
This is something that comes from the backend, we cannot really change it. 
If you look at the next line, it's used to create a camelCase variable. Does it seem fine to you?

Reply:
we could do qualified_conversation: qualifiedConversation, but its okay as we are assigning it to a variable afterwards